apps/builder/app/canvas/features/webstudio-component/selected-instance-connector.ts

```diff
@@ -113,6 +113,7 @@ export const SelectedInstanceConnector = ({
     selectedInstanceIntanceToTagStore.set(instanceToTag);
 
     return () => {
+      hideOutline();
```

Comment:
Can't this lead to race condition when other element is selected?

Reply:
Should not, as unsubscribe happens before subscribe

Reply:
But you batch events. Is there a guarantee sent events have the same order on the other side?